tests/e2e/gauntlet_deployer.go

```diff
@@ -0,0 +1,370 @@
+package e2e
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/url"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	. "github.com/onsi/gomega"
+	"github.com/smartcontractkit/chainlink-terra/tests/e2e/utils"
+	"github.com/smartcontractkit/integrations-framework/gauntlet"
+)
+
+const TERRA_COMMAND_ERROR = "Terra Command execution error"
+const RETRY_COUNT = 5
+
+type GauntletDeployer struct {
+	Cli                        *gauntlet.Gauntlet
+	Version                    string
+	LinkToken                  string
+	BillingAccessController    string
+	RequesterAccessController  string
+	Flags                      string
+	DeviationFlaggingValidator string
+	OCR                        string
+	RddPath                    string
+	ProposalId                 string
+	ProposalDigest             string
+}
+
+type InspectionResult struct {
+	Pass     bool
+	Key      string
+	Expected string
+	Actual   string
+}
+
+// GetDefaultGauntletConfig gets  the default config gauntlet will need to start making commands
+// 	against the environment
+func GetDefaultGauntletConfig(nodeUrl *url.URL) map[string]string {
+	networkConfig := map[string]string{
+		"NODE_URL":          nodeUrl.String(),
+		"CHAIN_ID":          "localterra",
+		"DEFAULT_GAS_PRICE": "1",
+		"MNEMONIC":          "symbol force gallery make bulk round subway violin worry mixture penalty kingdom boring survey tool fringe patrol sausage hard admit remember broken alien absorb",
+	}
+
+	return networkConfig
+}
+
+// UpdateReportName updates the report name to be used by gauntlet on completion
+func UpdateReportName(reportName string, g *gauntlet.Gauntlet) {
+	g.NetworkConfig["REPORT_NAME"] = filepath.Join(utils.Reports, reportName)
+	err := g.WriteNetworkConfigMap(utils.Networks)
+	Expect(err).ShouldNot(HaveOccurred(), "Failed to write the updated .env file")
+}
+
+// GetInspectionResultsFromOutput parses the inpsectiond data from the output
+//  TODO we should really update the inspection command to just output json in the future
```

Comment:
Can you open an issue/story with this? Maybe the json output could be the expected data vs. what we got onchain

Reply:
Created a story for this.